docs/concepts/save-query.asciidoc

```diff
@@ -28,18 +24,18 @@ For more information, see <<xpack-security-authorization, Granting access to Kib
 
 ==== Save a query
 
-. Once you’ve built a query worth saving, click the save query icon image:concepts/images/save-icon.png["save query icon"].
-. In the popover, click *Save current query*.
-. Enter a unique name to identify the query and an optional description that will appear in a tooltip in the saved query popover.
+. Once you’ve built a query worth saving, click the save query icon image:concepts/images/saved-query-icon.png[save query icon, width=24px].
+. In the menu, click *Save saved query*.
```

Comment:
This is a UI change, but **Save saved query** is strange. I know the query is already saved, but shouldn't it just be **Update saved query**?